apps/ledger-live-mobile/src/screens/OperationDetails/Content.tsx

```diff
@@ -144,7 +144,7 @@ export default function Content({
   const { EditOperationPanel: SpecificEditOperationPanel = undefined } =
     byFamiliesEditOperationPanel[
       mainAccount.currency.family as keyof typeof byFamiliesEditOperationPanel
```

Comment:
```suggestion
  const { EditOperationPanel: SpecificEditOperationPanel } =
    byFamiliesEditOperationPanel[
      mainAccount.currency.family as keyof typeof byFamiliesEditOperationPanel
```

`= undefined` is useless now no ?

Reply:
Not according to TS type inference apparently 🤔 

![Screenshot 2023-11-16 at 12 54 40](https://github.com/LedgerHQ/ledger-live/assets/9203826/9a76212f-462d-4896-a42b-ab37d2325bc6)
![Screenshot 2023-11-16 at 12 54 51](https://github.com/LedgerHQ/ledger-live/assets/9203826/dbe15548-e601-4fd4-b57a-b0b3575c939c)

Even if it might be redundant, I would rather keep it for better type inference and context. Similar to explicitly typing the return type of a function. Not _indispensable_ but provide more context